### PR TITLE
fix(HttpParser) always check if content length is valid before calling requestHandler  

### PR DESCRIPTION
**Pull request by cirospaciari**
### What does this PR do?

- [ ] Documentation or TypeScript types (it's okay to leave the rest blank in this case)
- [X] Code changes

### CODE REVIEW

**Comment by robobun** [comment]
@cirospaciari, your commit 072dc652eae375cdc49bcc3c7f64da3599dfd256 is building: [`#6245`](https://buildkite.com/bun/bun/builds/6245)

File: packages/bun-uws/src/HttpParser.h [hunk 1]
```diff
@@ -623,6 +625,15 @@ namespace uWS
             /* Parse query */
             const char *querySeparatorPtr = (const char *) memchr(req->headers->value.data(), '?', req->headers->value.length());
             req->querySeparator = (unsigned int) ((querySeparatorPtr ? querySeparatorPtr : req->headers->value.data() + req->headers->value.length()) - req->headers->value.data());
+            
+            // lets check if content len is valid before calling requestHandler
+            if(contentLengthStringLen) {
+                remainingStreamingBytes = toUnsignedInteger(contentLengthString);
+                if (remainingStreamingBytes == UINT64_MAX) {
```
Comment (Jarred-Sumner) [inline packages/bun-uws/src/HttpParser.h:632]:
```suggestion
                if (UNLIKELY(remainingStreamingBytes == UINT64_MAX)) {
```

File: packages/bun-uws/src/HttpParser.h [hunk 2]
```diff
@@ -613,7 +613,9 @@ namespace uWS
             * ought to be handled as an error. */
             std::string_view transferEncodingString = req->getHeader("transfer-encoding");
             std::string_view contentLengthString = req->getHeader("content-length");
-            if (transferEncodingString.length() && contentLengthString.length()) {
+            auto transferEncodingStringLen = transferEncodingString.length();
+            auto contentLengthStringLen = contentLengthString.length();
+            if (transferEncodingStringLen && contentLengthStringLen) {
```
Comment (Jarred-Sumner) [inline packages/bun-uws/src/HttpParser.h:618]:
```suggestion
            if (UNLIKELY(transferEncodingStringLen && contentLengthStringLen)) {
```

File: packages/bun-uws/src/HttpParser.h [hunk 3]
```diff
@@ -613,7 +613,9 @@ namespace uWS
             * ought to be handled as an error. */
             std::string_view transferEncodingString = req->getHeader("transfer-encoding");
             std::string_view contentLengthString = req->getHeader("content-length");
-            if (transferEncodingString.length() && contentLengthString.length()) {
+            auto transferEncodingStringLen = transferEncodingString.length();
```
Comment (Jarred-Sumner) [inline packages/bun-uws/src/HttpParser.h:616]:
```suggestion
            const auto transferEncodingStringLen = transferEncodingString.length();
```

File: packages/bun-uws/src/HttpParser.h [hunk 4]
```diff
@@ -613,7 +613,9 @@ namespace uWS
...
+            auto contentLengthStringLen = contentLengthString.length();
```
Comment (Jarred-Sumner) [inline packages/bun-uws/src/HttpParser.h:617]:
```suggestion
            const auto contentLengthStringLen = contentLengthString.length();
```

**Pull request merged**